front/chat/src/Components/Chat.svelte

```diff
@@ -90,13 +131,26 @@
                     <!-- chat users -->
                     <UsersList
                             usersListStore={defaultRoom().getPresenceStore()}
-                            showUsers={showUsers}
+                            meStore={defaultRoom().getMeStore()}
+                            {showUsers}
                             searchValue={searchValue.toLocaleLowerCase()}
                             on:activeThread={handleActiveThread}
                             on:showUsers={handleShowUsers}
-                            on:goTo={(event) => defaultRoom().goTo(event.detail.type, event.detail.roomId, event.detail.uuid)}
+                            on:goTo={(event) => defaultRoom().goTo(event.detail.type, event.detail.playUri, event.detail.uuid)}
+                            on:rankUp={(event) => defaultRoom().rankUp(event.detail.jid)}
+                            on:rankDown={(event) => defaultRoom().rankDown(event.detail.jid)}
+                            on:ban={(event) => defaultRoom().ban(event.detail.user, event.detail.name, event.detail.playUri)}
+                    />
+
+                    <ChatLiveRooms
+                            {showLives}
+                            searchValue={searchValue.toLocaleLowerCase()}
+                            on:activeThread={handleActiveThread}
+                            on:showLives={handleShowLives}
+                            liveRooms={[...$mucRoomsStore].filter(mucRoom => mucRoom.type === 'live' && mucRoom.name.toLowerCase().includes(searchValue))}
```

Comment:
Why use `liveRooms` export when you can use `mucRoomsStore` directly in the component `ChatLiveRooms`?